guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -133,13 +133,14 @@ ifdef::katello[]
 # {foreman-installer}
 ----
 endif::[]
-. Check when the kernel packages were last updated:
+. Check if the newest installed kernel is currently running:
 +
 [options="nowrap"]
 ----
-# rpm -qa --last | grep kernel
+# rpm --query --last kernel | head -n 1 | grep $(uname --kernel-release)
 ----
-. Optional: If a kernel update occurred since the last reboot, reboot the system:
+If this command produces the kernel version as output, it means that the newest installed kernel is running. Otherwise the system needs a reboot to run the newest kernel.
```

Comment:
```suggestion
If this command produces the kernel version as output, it means that the newest installed kernel is running.
Otherwise the system needs a reboot to run the newest kernel.
```